### ISSUE TITLE
Doesn't work with Bukkit v1.5

### ISSUE THREAD

**Issue by RENMX** [Sorroko/HTTPConsole#6]
The current release and the community fixes releases do not work with Minecraft 1.5. The community modified version does run but certain commands such as /lag and /who sent through httpconsole receive an error response instead of the correct information.

A fix would be greatly appreciated.

**Comment by Sorroko** [comment]
Thank you for the report, testing a fix now. Is it all commands or just some?

**Comment by Sorroko** [comment]
Didn't mean to close the issue!

**Comment by RENMX** [comment]
When I tested it, all of the vanilla commands I tried did not work. I tried "who", "lag" and "op". Plugin based commands however such as pex reload (to reload the permissionex config file) work fine.

When vanilla commands fail to work this is the message sent back from httpconsole: "Error: Invalid parameters" that happens on op, who and lag even though the command does get executed. For example when I send lag, I can see the tick, chunk and entity information appear inside the games console it just isn't being sent through httpconsole and instead I receive that error I noted above.
